src/components/CreateEvent/CreateEvent.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { useState } from 'react';
+import { CreateEventWrapper, Form } from './styled';
+import { parseISO, formatISO, format } from 'date-fns';
+
+
+function CreateEvent() {
+  const [name, setName] = useState("");
+  const [image, setImage] = useState("");
+  const [imageAlt, setImageAlt] = useState("");
+  // const [date, setDate] = useState(Date.now());
+  const [time, setTime] = useState(formatISO(Date.now()));
+  const [description, setDescription] = useState("");
+  const [eventType, setEventType] = useState("");
+  const [firstName, setFirstName] = useState("");
+  const [lastName, setLastName] = useState("");
+  const [email, setEmail] = useState("");
+  
+  function create () {
+    // TODO fix url in env variable
+    fetch("https://localhost:5001/api/Event", {
+      method: "POST", // *GET, POST, PUT, DELETE, etc.
+      mode: "cors",
+    headers: {
+        "Content-Type": "application/json",
+    },
+      body: JSON.stringify({
+        id: "5f6b241978964f06faa3185b",
+        time: time,
+        name: name,
+        description: description,
+        owner: {
+          id: "5f6b242580090096cfb631a1",
+          firstName: firstName,
+          lastName: lastName,
+          email: email
+        },
+        participants: [{
+          id: "5f6b242580090096cfb631a1",
+          firstName: firstName,
+          lastName: lastName,
+          email: email
+        }]
+      })
+    })
+    .then(() => {
+      console.log('Event created');
+    })
+    .catch(error => { console.log(error)})
+  }
+
+  function isFormFilled () {
+    if (name !== "" && description !== "" && eventType !== "") {
+      return true
+    } else {
+      return false
+    }
+  }
```

Comment:
Utility functions like these should typically reside outside of the component. There are two reasons for this:

1. readability; and
2. avoiding unnecessary re-renders.

This case is not affected by 2., but that's it still something worth having in mind when writing React components.